setup.cfg

```diff
@@ -48,7 +62,8 @@ exclude =
 # PDF = ReportLab; RXP
 # Add here test requirements (semicolon/line-separated)
 testing =
-    pytest
+    pytest==5.0.1
```

Comment:
pytest is pinned for now due to some windows issue https://github.com/pytest-dev/pytest/issues/5826
But I think this should be fixed soon